WalletWasabi.Fluent/Models/Wallets/WalletSettingsModel.cs

```diff
@@ -42,17 +44,20 @@ public WalletSettingsModel(KeyManager keyManager, bool isNewWallet = false, bool
 		_redCoinIsolation = _keyManager.RedCoinIsolation;
 		_coinjoinSkipFactors = _keyManager.CoinjoinSkipFactors;
 		_feeRateMedianTimeFrameHours = _keyManager.FeeRateMedianTimeFrameHours;
+		_outputWalletId = Services.WalletManager.GetWalletByName(_keyManager.WalletName).WalletId;
 
 		WalletType = WalletHelpers.GetType(_keyManager);
 
 		this.WhenAnyValue(
-			x => x.AutoCoinjoin,
-			x => x.IsCoinjoinProfileSelected,
-			x => x.PreferPsbtWorkflow,
-			x => x.PlebStopThreshold,
-			x => x.AnonScoreTarget,
-			x => x.RedCoinIsolation,
-			x => x.FeeRateMedianTimeFrameHours)
+				x => x.AutoCoinjoin,
+				x => x.IsCoinjoinProfileSelected,
+				x => x.PreferPsbtWorkflow,
+				x => x.PlebStopThreshold,
+				x => x.AnonScoreTarget,
+				x => x.RedCoinIsolation,
+				x => x.FeeRateMedianTimeFrameHours,
+				x => x.OutputWalletId,
```

Comment:
WalletID is not saved in file, so it's change doesn't have to call `SetValues()`. You can revert.

Reply:
Done